### PR TITLE
🔨 (docker) switch to Docker `compose` command

### PR DESCRIPTION
**Pull request by cuchulainn-fun**
## Purpose

The bootstrap of Potsie works with the docker compose current version

## Proposal

Change the reference docker compose with the dash to docker compose without the dash in the files

### CODE REVIEW

**Comment by quitterie-lcs** [comment]
We were using ubuntu image 20.04 (from April 2021) for Linux machine environment in `lint` and `plugins` jobs. Only docker-compose` was installed. In recent release, they introduce [Docker Compose Switch](https://github.com/docker/compose-switch) to translate commands into Compose V2 and then understand `docker compose` command. I update the ubuntu image tags to the latest released: `2023.02.1`

File: .dockerignore [hunk 1]
```diff
@@ -12,7 +12,7 @@ build
 **/.DS_Store
 
 # Docker
-docker-compose.*
+docker compose.*
```
Comment (jmaupetit) [inline .dockerignore:15]: You don't need to update this file that reference file names not commands to execute.

**Review by jmaupetit — COMMENTED** [review]
Thank you for this! :muscle: :sunglasses: 

I recommend:

- updating your git commit title to ":hammer: (docker) switch to Docker 'compose' command"
- updating your commit message to something like:

> Docker compose is now distributed as a `docker` program command, let's switch to it!

File: .dockerignore (hunk 1 above)
Comment (cuchulainn-fun) [reply .dockerignore:15]: Thank you for your suggestion and explanation :) 
I'll update my commit that as soon as possible

**Comment by quitterie-lcs** [comment]
@cuchulainn-fun I have restored the fix commit, you just have to update the .dockerignore file accord to Julien's review and we will be ready to validate your PR

**Comment by cuchulainn-fun** [comment]
Thank you @quitterie-lcs I will do what is necessary and my best

**Review by quitterie-lcs — APPROVED** [review]

**Review by wilbrdt — APPROVED** [review]
Alt text

**Pull request merged**